qemu/tests/cfg/netkvm_log_check.cfg

```diff
@@ -0,0 +1,12 @@
+- netkvm_log_check:
```

Comment:
Could you explain why we name the case `netkvm_log_check`? My understanding of this case is about TXcapacity function, netkvm_log_check is too general and seems not what this case is really about.

Reply:
Yes, this case is only about "TXcapacity" for now. However, our developer may wish to add more keywords to the feature. This is why I gave it this common name and expanded its range.

Reply:
`netkvm_params_check` would makes more sense. Log check is the step but not the test object. Also then we better explicitly clarify TXcapacity in the test, consider `netkvm_params_check.tx_capacity`, then you need a variants defined.